### PR TITLE
updated README.md

### PR DESCRIPTION
**Pull request by danielemonaca**
I have seen different Issues having the same mistake of not setting the required css, so I tried to make it a little clearer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,7 +100,15 @@ XLM Wallet Address:
 - `npm install ngx-quill`
 - for projects using Angular < v5.0.0 install `npm install ngx-quill@1.6.0`
 - install `@angular/core`, `@angular/common`, `@angular/forms`, `@angular/platform-browser`, `quill`, and `rxjs` - peer dependencies of ngx-quill
-- include theme stylings: bubble.css, snow.css of quilljs in your index.html, or add them in your css/scss files with `@import` statements, or add them external stylings in your build process.
+- include theme stylings: 	**bubble.css, snow.css and core.css of quilljs** in your index.html (you can find them in `node_modules/dist`), or add them in your css/scss files  with `@import` statements, or add them external stylings in your build process.
```
Comment (KillerCodeMonkey) [inline README.md:103]: > you can find them in `node_modules/dist`

should be `node_modules/quill/dist`?

**Review by KillerCodeMonkey — COMMENTED** [review]
check my comment and thanks :)

File: README.md (hunk 1 above)
Comment (danielemonaca) [reply README.md]: Yep! Sorry for that, I pushed the change 👍

**Comment by KillerCodeMonkey** [comment]
Thanks will merge it tomorrow

Daniele Monaca <notifications@github.com> schrieb am Mo., 31. Aug. 2020,
17:19:

**Pull request merged**